rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
```

Comment:
We may want to use `dist-upgrade` instead. Usually, not my go to command but given this is the base image it seems appropriate since this is not being run on a production/live server(s).

https://askubuntu.com/questions/194651/why-use-apt-get-upgrade-instead-of-apt-get-dist-upgrade does a decent job at explaining the difference for those curious